tests/e2e/forms/family-survey-form.specs.js

```diff
@@ -38,11 +36,11 @@ describe('Family Survey form', () => {
     familyForm.fillFamilySurvey(1, 3);
     genericForm.nextPage();
     familyForm.familyConditions(1);
+    genericForm.cancel();
```

Comment:
I don't understand how this works. Is it cancelling the form? If so, how can we submit it on the very next line? Is it actually cancelling something else?

Reply:
The cancel was because the rest of the tests on this spec are not valid based on the resulting report output.I've resubmitted changes for review